robottelo/ui/base.py

```diff
@@ -194,7 +194,11 @@ def delete(self, name, really=True, dropdown_present=False,
                search_query=None):
         """Delete an added entity, handles both with and without dropdown."""
         self.logger.debug(u'Deleting entity %s', name)
-        searched = self.search(name, _raw_query=search_query)
+        # Overridden search methods do not support search queries
```

Comment:
so: it must be as:
```console
# Note: Some overridden search methods do not support search queries
```
as if overridden search methods do not support search queries, we do need this PR :) than